releasenotes/notes/user-defined-git-metadata-c333300e16662551.yaml

```diff
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    It is now possible to specify any of the following git metadata through environment variables:
```

Comment:
We should specify this is for CI App/pytest plugin.

I don't want someone to think they can do this for all spans/services/etc